src/NexusMods.App/Commandline/CleanupVerbs.cs

```diff
@@ -0,0 +1,180 @@
+                DataModelSettings.GetStandardDataModelFolder(fileSystem)
+            }.Concat(dataModelSettings.ArchiveLocations.Select(path => path.ToPath(fileSystem)));
+
+            if (fileSystem.OS.IsUnix())
+                await DeleteRemainingFilesUnix(renderer, appFiles, appDirectories);
+            else
+                DeleteRemainingFilesWindows(appFiles, appDirectories);
+
+            await renderer.Text("Application uninstalled successfully");
+            return 0;
+        }
+        catch (Exception ex)
+        {
+            await renderer.Error(ex, $"Error deleting application directories: {ex.Message}");
+            return -1;
+        }
+        finally
+        {
+            Environment.Exit(0);
+        }
+    }
+
+    private static async Task DeleteRemainingFilesUnix(IRenderer renderer, AbsolutePath[] appFiles, IEnumerable<AbsolutePath> appDirectories)
+    {
+        await DeleteFilesUnix(renderer, appFiles);
+        await DeleteDirectoriesUnix(renderer, appDirectories);
+    }
+    
+    private static void DeleteRemainingFilesWindows(AbsolutePath[] appFiles, IEnumerable<AbsolutePath> appDirectories)
+    {
+        var scriptContent = new StringBuilder();
+
+        // Kill the App Client and Server
+        scriptContent.AppendLine("Stop-Process -Name \"NexusMods.App\" -Force -ErrorAction SilentlyContinue");
+
+        // Note(Sewer) Ensure the process handles are freed, just in case.
+        scriptContent.AppendLine("Start-Sleep -Seconds 1");
+
+        foreach (var appFile in appFiles)
+            scriptContent.AppendLine($"Remove-Item -Path \"{appFile}\" -Force -ErrorAction SilentlyContinue");
+
+        foreach (var directory in appDirectories)
+            scriptContent.AppendLine($"Remove-Item -Path \"{directory}\" -Recurse -Force -ErrorAction SilentlyContinue");
+
+        // Remove the script itself
+        scriptContent.AppendLine("Remove-Item -Path $MyInvocation.MyCommand.Path -Force -ErrorAction SilentlyContinue");
+
+        // Save the script to a temporary file
+        var tempFile = Path.GetTempFileName();
+        var tempScriptPath = tempFile.Replace(".tmp", ".ps1");
+        File.Delete(tempFile);
+        File.WriteAllText(tempScriptPath, scriptContent.ToString());
+
+        // Execute the script
+        Task.Run(async () => await Cli.Wrap("powershell.exe")
+            .WithArguments(new[] { "-ExecutionPolicy", "Bypass", "-File", tempScriptPath })
+            .ExecuteBufferedAsync()).Wait();
+
+        // Script will terminate this process, this is just ensuring we don't exit early.
+        Thread.Sleep(10);
+        File.Delete(tempScriptPath);
+    }
+
+    private static async Task DeleteDirectoriesUnix(IRenderer renderer, IEnumerable<AbsolutePath> appDirectories)
+    {
+        foreach (var directory in appDirectories)
+        {
+            if (!directory.DirectoryExists())
+                continue;
+
+            directory.Delete();
```

Comment:
`AbsolutePath.Delete` calls `IFileSystem.DeleteFile`. This should be `IFileSystem.DeleteDirectory`.

This should also be wrapped in a try/catch block.